dashboard/src/components/Config/PkgRepoList/PkgRepoForm.tsx

```diff
@@ -0,0 +1,1701 @@
+                        id="kubeapps-repo-filter-repositories"
+                        placeholder={"nginx, jenkins"}
+                        value={filterNames}
+                        onChange={handleFilterNamesChange}
+                      />
+                    </CdsTextarea>
+                    <CdsCheckbox className="ca-skip-tls">
+                      <label htmlFor="kubeapps-repo-filter-exclude">Exclude Packages</label>
+                      <CdsControlMessage>
+                        Exclude packages matching the given filter
+                      </CdsControlMessage>
+                      <input
+                        id="kubeapps-repo-filter-exclude"
+                        type="checkbox"
+                        onChange={handleFilterExcludeChange}
+                        checked={filterExclude}
+                      />
+                    </CdsCheckbox>
+                    <CdsCheckbox className="ca-skip-tls">
+                      <label htmlFor="kubeapps-repo-filter-regex">Regular Expression</label>
+                      <CdsControlMessage>
+                        Mark this box to treat the filter as a regular expression
+                      </CdsControlMessage>
+                      <input
+                        id="kubeapps-repo-filter-regex"
+                        type="checkbox"
+                        onChange={handleFilterRegexChange}
+                        checked={filterRegex}
+                      />
+                    </CdsCheckbox>
+                  </>
+                )}
+              </CdsFormGroup>
+            </CdsAccordionContent>
+          </CdsAccordionPanel>
+          <CdsAccordionPanel id="panel-advanced" expanded={accordion[3]}>
+            <CdsAccordionHeader onClick={() => toggleAccordion(3)}>Advanced</CdsAccordionHeader>
+            <CdsAccordionContent>
+              <CdsFormGroup layout="vertical">
+                {plugin?.name !== PluginNames.PACKAGES_HELM && (
+                  <CdsInput>
+                    <label htmlFor="kubeapps-repo-interval">Synchronization Interval</label>
+                    <input
+                      id="kubeapps-repo-interval"
+                      type="text"
+                      placeholder="10m"
+                      value={interval}
+                      onChange={handleIntervalChange}
+                    />
+                    <CdsControlMessage>
+                      Time (expressed as a{" "}
+                      <a
+                        href={"https://pkg.go.dev/time#ParseDuration"}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        Golang duration
+                      </a>
+                      ) to wait between synchronizing the repository.
+                    </CdsControlMessage>
+                  </CdsInput>
+                )}
+                {plugin?.name === PluginNames.PACKAGES_HELM && (
```

Comment:
Looking forward to the Plugins UI customization to be implemented to avoid this kind of condition blocks!